### PR TITLE
Added more detail to Visual Studio compile markdown files

### PR DESCRIPTION
**Pull request by KarateSnoopy**
#### Summary
SUMMARY: Infrastructure "Added more detail to Visual Studio compile markdown files"

#### Purpose of change
Added missing detail about precompile.cmd and added link to compile.md from vc-pkg guide.

### CODE REVIEW

File: COMPILING.md [hunk 1]
```diff
@@ -567,7 +567,7 @@ Extract the 'WinDepend' folder and put it in the root folder of Cataclysm projec
 
 ### Building
 
-Building Cataclysm with Visual Studio is very simple. Just build it like a normal Visual C++ project. The process may takes a long period of time, so you'd better prepare a cup of coffee and some books in front of your computer :)
+Building Cataclysm with Visual Studio is very simple. First create the version.h file by running the batch script `prebuild.cmd` in the folder 'msvc-full-features'.  Then just build it like a normal Visual C++ project. The process may takes a long period of time, so you'd better prepare a cup of coffee and some books in front of your computer :)
```
Comment (ZhilkinSerg) [inline COMPILING.md]: I am pretty sure that `prebuild.cmd` is run automatically as prebuild event:

https://github.com/CleverRaven/Cataclysm-DDA/blob/95bfcf2380a8a1e1a2865db17bf3b397bd07c6d9/msvc-full-features/Cataclysm-vcpkg-static.vcxproj#L155-L157

Comment (KarateSnoopy) [reply COMPILING.md]: Hmm, I got a compile error and was missing version.h when I built it in VS2017

Comment (ZhilkinSerg) [reply COMPILING.md]: Could be some user side settings. I will need to check that.

Comment (ZhilkinSerg) [reply COMPILING.md]: It seems `prebuild.cmd` wasn't set in prebuild event only for Win32, but not for x64:

image

image

Comment (ZhilkinSerg) [reply COMPILING.md]: Also you need to have `git` available on some `PATH` as version string is generated by following command:

https://github.com/CleverRaven/Cataclysm-DDA/blob/95bfcf2380a8a1e1a2865db17bf3b397bd07c6d9/msvc-full-features/prebuild.cmd#L8

I will update the script to generate some version number without git.

Comment (KarateSnoopy) [reply COMPILING.md]: @ZhilkinSerg Thanks.  I'll get rid of this comment then

Comment (ZhilkinSerg) [reply COMPILING.md]: Try the files from #29070 and see if it helps.

Comment (KarateSnoopy) [reply COMPILING.md]: Will do.  I updated this PR with just a link to compiling.md

**Pull request merged**